### DISCUSSION TITLE
(練習)請問這次的課程講義要去哪裡下載？

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS4719-C/Python-Community-GS4719-C#366] [Q&A] [answered]
# Hello AI

請問這次的 _課程講義_ 要去哪裡**下載**？

![image](https://user-images.githubusercontent.com/86505454/153531482-f485a3b7-2c00-4735-9447-eb0fbeb2b8f5.png)

**Comment by chia-kai-chang** [comment, marked as answer]
「AI人工智慧導論」的Github網址在這邊喔：https://github.com/NCU-GS3073-AI-Introduction
